pkg/networkserver/networkserver.go

```diff
@@ -198,7 +198,7 @@ func New(c *component.Component, conf *Config, opts ...Option) (*NetworkServer,
 	if len(devAddrPrefixes) == 0 {
 		devAddr, err := types.NewDevAddr(conf.NetID, nil)
 		if err != nil {
-			return nil, err
+			panic(errors.New("failed to create new DevAddr").WithCause(err))
```

Comment:
Why?

This may come from config; this is not an implementation issue necessarily. Just return an error

Reply:
removed